test/wrapper_array.jl

```diff
@@ -1,18 +1,24 @@
 using NamedDims
-using NamedDims: names
 using SparseArrays
 using Test
 
 
 @testset "get the parent array that was wrapped" begin
     for orig in ([1 2; 3 4], spzeros(2, 2))
         @test parent(NamedDimsArray(orig, (:x, :y))) === orig
-
         @test unname(NamedDimsArray(orig, (:x, :y))) === orig
         @test unname(orig) === orig
     end
 end
 
+@testset "function dim(array, dim)" begin
+    nda = NamedDimsArray([10 20; 30 40], (:x, :y))
+    @test dim(nda, :x) == 1
+    @test dim(nda, [:y, :x]) == [2, 1]
+    @test dim(nda, 2) == 2
+    @test dim(parent(nda), 2) == 2
+    @test_throws ArgumentError dim(nda, :nope)
```

Comment:
I am surprised this is an `ArgumentError`, don't we normally use something more specific?

Reply:
It's from here:

https://github.com/invenia/NamedDims.jl/blob/master/src/name_core.jl#L45

Reply:
We should change that one day.
But not today